frontend/dispatcher.lua

```diff
@@ -1,3 +1,29 @@
+--[[--
+This module is responsible for dispatching events.
+
+To add a new action an entry must be added to `settingsList` & `dispatcher_menu_order`
+This can also be done at runtime via @{registerAction}.
```

Comment:
I prefer it like this.
```suggestion
This can also be done at runtime via @{registerAction}().
```